tests/cypress/integration/actions_projects_models_organizations/registration_involved/case_113_new_organization_pipeline.js

```diff
@@ -0,0 +1,284 @@
+            secondUser.lastName,
+            secondUserName,
+            secondUser.emailAddr,
+            secondUser.password,
+        );
+        cy.logout(secondUserName);
+
+        cy.goToRegisterPage();
+        cy.userRegistration(
+            thirdUser.firstName,
+            thirdUser.lastName,
+            thirdUserName,
+            thirdUser.emailAddr,
+            thirdUser.password,
+        );
+        cy.logout(thirdUserName);
+
+        cy.goToRegisterPage();
+        cy.userRegistration(
+            firstUser.firstName,
+            firstUser.lastName,
+            firstUserName,
+            firstUser.emailAddr,
+            firstUser.password,
+        );
+    });
+
+    beforeEach(() => {
+        Cypress.Cookies.preserveOnce('sessionid', 'csrftoken');
+    });
+
+    after(() => {
+        cy.logout(thirdUserName);
+        cy.getAuthKey().then((response) => {
+            cy.deletingRegisteredUsers(response, [firstUserName, secondUserName, thirdUserName]);
+            cy.deletingCreatedTasks(response, [newTaskName]);
+            cy.deletingCreatedProjects(response, [project.name]);
+            cy.deletingCreatedOrganizations(response, [organizationParams.shortName]);
+        });
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('The first user creates an organization and activates it.', () => {
+            cy.createOrganization(organizationParams);
+            cy.activateOrganization(organizationParams.shortName);
+        });
+
+        it('Open the organization settings. Invite members.', () => {
+            cy.openOrganization(organizationParams.shortName);
+            cy.checkOrganizationParams(organizationParams);
+            cy.checkOrganizationMembers(1, [firstUserName]);
+            const membersToInvite = [
+                {
+                    email: capitalizeEmail(secondUser.emailAddr),
+                    role: 'Worker',
+                },
+                {
+                    email: thirdUser.emailAddr,
+                    role: 'Worker',
+                },
+            ];
+            cy.inviteMembersToOrganization(membersToInvite);
+            cy.checkOrganizationMembers(3, [firstUserName, secondUserName, thirdUserName]);
+        });
+
+        it('Create a project, create a task. Deactivate organization.', () => {
+            cy.goToProjectsList();
+            cy.createProjects(
+                project.name,
+                project.label,
+                project.attrName,
+                project.attrVaue,
+                project.multiAttrParams,
+            );
+            cy.goToTaskList();
+            cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
+            cy.openTask(taskName);
+            cy.assignTaskToUser(secondUserName);
+            cy.activateOrganization('Personal workspace'); // Deactivate organization
+        });
+
+        // FIXME: Activate after implementation
```

Comment:
Why do you have fixme here?

Reply:
This step was skipped due to a different behavior than indicated in the script. The user sees the project and task outside the organization because he created them. We talked about it once.

Reply:
Yep, got it. This bug will be fixed soon.